Partial_Castor_edits.py

```diff
@@ -0,0 +1,53 @@
+# Capture .ser files with single.png for "ser_time" seconds at 1000x1000 resolution
+#
+# Uses user settings for exposure and gain
+# Forces resolution, binning, colour space

```

Comment:
I think it'd be better if we can figure out how to use sharpcap "profiles"?  That will cut down on the amount of code.